docker/configure_workers_and_start.py

```diff
@@ -339,7 +340,7 @@ def generate_worker_files(environ, config_path: str, data_dir: str):
     # base shared worker jinja2 template.
     #
     # This config file will be passed to all workers, included Synapse's main process.
-    shared_config = {"listeners": listeners}
+    shared_config: dict = {"listeners": listeners}
```

Comment:
```suggestion
    shared_config: Dict[str, Any] = {"listeners": listeners}
```

Reply:
I can't remember why I had to force the type of the variable. But now, the type from the value assigned does not produce any Mypy error in any line that uses `shared_config.` so I have removed it.